### PR TITLE
e-signature signing page tweaks

### PR DESCRIPTION
**Pull request by gidsg**
Another couple of fixes from: https://trello.com/c/NutOcwCP/342-2-placeholder-triage-and-fix-e-signature-issues
* Updated error message suggested by @NoraGDS 
* Make form two thirds column
Screenshot 2020-09-30 at 13 57 39

### CODE REVIEW

**Review by bjgill — APPROVED** [review]
:shipit:

**Comment by domoscargin** [comment]
Could we wrap the entire page in two-thirds? Most other pages are.

File: app/templates/frameworks/sign_framework_agreement.html [hunk 1]
```diff
@@ -29,6 +29,8 @@
 
 {% block mainContent %}
     {% include 'toolkit/forms/validation.html' %}
+    <div class="govuk-grid-row">
```
Comment (domoscargin) [inline app/templates/frameworks/sign_framework_agreement.html:32]: 👍

**Review by domoscargin — APPROVED** [review]
Awesome!

**Pull request merged**